extensions/accessibilityExtension/README.md

```diff
@@ -0,0 +1,262 @@
+**type**    | type IRI array |`['Extension', 'extensions:Accessibility']`
+**accessibilityAPI** | text | Indicates that the resource is compatible with the referenced accessibility API. Possible values: `['AndroidAccessibility', 'ARIA', 'ATK', 'AT-SPI', 'BlackberryAccessibility', 'iAccessible2', 'iOSAccessibility', 'JavaAccessibility', 'MacOSXAccessibility', 'MSAA', 'UIAutomation']`
+**accessibilityControl** | text | Identifies one or more input methods that allow access to all of the application functionality. Possible values: `['fullKeyboardControl', 'fullMouseControl', 'fullSwitchControl', 'fullTouchControl', 'fullVideoControl', 'fullVoiceControl']`
+**accessibilityFeature** | text | Content features of the resource, such as accessible media, supported enhancements for accessibility and alternatives. `['alternativeText', 'annotations', 'audioDescription', 'bookmarks', 'braille', 'captions', 'ChemML', 'describedMath', 'displayTransformability', 'highContrastAudio', 'highContrastDisplay', 'index', 'largePrint', 'latex', 'longDescription','MathML', 'none', 'printPageNumbers', 'readingOrder', 'signLanguage', 'structuralNavigation', 'tableOfContents', 'taggedPDF', 'tactileGraphic', 'tactileObject', 'timingControl', 'transcript', 'ttsMarkup', 'unlocked']`
+**accessibilityHazard** | text | A characteristic of the described resource that is physiologically dangerous to some users. Related to http://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure.html `['flashing', 'noFlashingHazard', 'motionSimulation', 'noMotionSimulationHazard', 'sound', 'noSoundHazard']`
+
+**Extendable Badge Objects:**
+Assertion, BadgeClass, Issuer
+
+**Example implementation:**
+```json
+{
+  "name": "Awesome Robotics Badge",
+  "description": "For doing awesome things with robots that people think is pretty great.",
+  "image": "http://openbadges.it/logo.png",
+  "criteria": "https://example.org/robotics-badge.html",
+  "tags": [
+      "robots",
+      "awesome"
+  ],
+  "issuer": "https://example.org/badge/issuer.json",
+  "extensions:Accessibility": {
+    "@context":"https://w3id.org/openbadges/extensions/accessibilityExtension/context.json",
+    "type": ["Extension", "extensions:Accessibility"],
+    "accessibilityAPI": "ARIA",
+    "accessibilityControl": ["fullKeyboardControl","fullMouseControl","fullTouchControl"],
+    "accessibilityFeature": "audioDescription",
+    "accessibilityHazard": "noFlashingHazard",
+    "url": "http://exampleaccessiblecontent.org/"
+  }
+}
+```
+
+## Open Badges 3.0
+
+The adaptation of this extension ultimately consist on extending the
+datamodel, adding existing accessibility attributes to the entities
+`Achievement`, `AchievementCredential` and `Profile`.
+
+The OB 3.0 JSON-LD context has already defined some terms that this
+extension to all entities of the specification. So, you'll need a JSON-LD
+context with the remaining terms.
+
+```json
+// Accessibility OB 3.0 JSON-LD Context
```

Comment:
Can you create a caption for this outside of the snippet?

Reply:
Moved all comments outside the snippet and replaced by captions